.changeset/early-weeks-watch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': major
+---
+
+Removed `data-iui-scroll-placement` and `data-iui-overflow` attributes.
```

Comment:
react user won't care, so this changeset can be removed

Reply:
Removed